app/experimenter/features/management/commands/load_feature_configs.py

```diff
@@ -35,4 +45,10 @@ def handle(self, *args, **options):
             feature_config.save()
             logger.info(f"Feature Loaded: {feature.applicationSlug}/{feature.slug}")
 
+        for feature_slug in all_features_in_db:
+            logger.warning(f"Feature Not Found in YAML: {feature_slug}")
```

Comment:
Tiny nit: all of the other logs in this are `.info`, maybe change this one to be consistent?